shared/types/src/dataset.ts

```diff
@@ -690,6 +690,7 @@ type SelectCohortEntry = {
 	values: SelectCohortValuesEntry[]
 	description?: string
 	asterisk?: string
+	clearOnChange?: { [index: string]: boolean }
```

Comment:
thanks. please move the comments here to explain this property

Reply:
Done

Reply:
todo later: it would be better to define specific names for the allowed clearOnChange property names, so that invalid props would be typechecked, but this is  not urgent, can discuss and do later

```ts
clearOnChange?: { plots?: true, filter?: true }
```